README.md

```diff
@@ -200,3 +200,12 @@ If you have disabled the "admin party" in CouchDB, you might have to manually sp
 ```shell
 $ COUCHDB_URL='http://admin:admin@127.0.0.1:5984/' bundle exec rake test
 ```
+
+## Releasing a new version
+
+- If you have never released a version of this gem before: `git remote add fury https://ecraft-gems@git.fury.io/ecraft-gems/couch_tap.git`
+- Merge all relevant pull requests
+- Bump the version in the `VERSION` file. Follow Semantic Versioning principles.
```

Comment:
Don't push this, since the next step pushes automatically.

Also, the version in `VERSION` does not include the leading `v`